alerts.tf

```diff
@@ -73,4 +73,24 @@ module "div-bulkcase-errors-alert" {
   trigger_threshold_operator = "GreaterThan"
   trigger_threshold = 0
   resourcegroup_name = "${azurerm_resource_group.rg.name}"
+}
+
+module "div-data-extraction-alert" {
+  source = "git@github.com:hmcts/cnp-module-metric-alert"
+  location = "${var.location}"
+
+  app_insights_name = "div-${var.env}"
+
+  alert_name = "Data extraction to Family Man has not happened yet"
```

Comment:
As far as I can remember, alert names cannot have spaces (only `-`) - see other alert names
I'm trying to figure out how we used to test these alerts in sandbox, but for some reason I cannot see the branch in https://sandbox-build.platform.hmcts.net/job/HMCTS_Sandbox_DIV/job/div-shared-infrastructure/

Reply:
Maybe if you remove `enabled` and rebuild https://sandbox-build.platform.hmcts.net/job/HMCTS_Sandbox_DIV/job/div-shared-infrastructure/view/change-requests/job/PR-55/ it might create it in div-sandbox appinsights

Reply:
Done and done.